sphinx/domains/std.py

```diff
@@ -437,11 +439,18 @@ class ProductionList(SphinxDirective):
     def run(self) -> List[Node]:
         domain = cast(StandardDomain, self.env.get_domain('std'))
         node = addnodes.productionlist()  # type: Element
+        # The backslash handling is from ObjectDescription.get_signatures
+        nl_escape_re = re.compile(r'\\\n')
+        strip_backslash_re = re.compile(r'\\(.)')
+        lines = nl_escape_re.sub('', self.arguments[0]).split('\n')
+        # remove backslashes to support (dummy) escapes; helps Vim highlighting
```

Comment:
Is this still useful for vim? I don't know this is still needed.

Reply:
I don't know, but I am all for simplifying it if possible. The code was copied from ``ObjectDescription``.

Reply:
It's not good to add not understood code. This code is strange to me. How about removal? It's not too late to add it after if needed.

Reply:
Indeed, not need to introduce it when it wasn't there before. It has been removed.